config/internal/instascale/clusterrole.yaml.tmpl

```diff
@@ -21,6 +21,23 @@ rules:
   - delete
   - patch
 
+- apiGroups:
```

Comment:
I think that's an acceptable solution to fix the issue in the short term. However, I think the management of the OCM secret should be reviewed. I think it should be exposed in the InstaScale API as an object reference, and the RBAC configured accordingly. If that makes sense, could you please open an issue?

Reply:
Issue created and left in backlog. Thank you. https://github.com/project-codeflare/instascale/issues/87